themes/bootstrap3/templates/Recommend/SideFacets/single-facet.phtml

```diff
@@ -33,30 +33,48 @@
 <?php if ($hasSubLinks): ?>
   <div class="<?=implode(' ', $classList) ?>">
 <?php else: ?>
-  <a href="<?=$toggleUrl ?>" class="<?=implode(' ', $classList) ?>" data-title="<?=$this->escapeHtmlAttr($this->facet['displayText']) ?>" data-count="<?=$this->facet['count'] ?>"<?php if($this->facet['isApplied']): ?> title="<?=$this->transEscAttr('applied_filter') ?>"<?php endif;?> data-lightbox-ignore>
+  <a href="<?=$toggleUrl ?>"
+     class="<?=implode(' ', $classList) ?>"
+     data-title="<?=$this->escapeHtmlAttr($this->facet['displayText']) ?>"
+     data-count="<?=$this->facet['count'] ?>"
+     title="<?php if($this->facet['isApplied']): ?><?=$this->transEsc('applied_filter') ?> - <?=$this->transEsc('page_reload_on_deselect_hint', ['%%filter_name%%' => $this->facet['displayText']])?><?php else: ?><?=$this->transEsc('page_reload_on_select_hint', ['%%filter_name%%' => $this->facet['displayText']])?>
+     <?php endif;?>"
+     data-lightbox-ignore>
 <?php endif; ?>
 
   <?php if ($hasSubLinks): ?>
-    <a class="text" href="<?=$toggleUrl ?>" data-lightbox-ignore data-title="<?=$this->escapeHtmlAttr($this->facet['displayText']) ?>" data-count="<?=$this->facet['count'] ?>"<?php if($this->facet['isApplied']): ?> title="<?=$this->transEscAttr('applied_filter') ?>"<?php endif;?>>
+    <a class="text" href="<?=$toggleUrl ?>" data-lightbox-ignore data-title="<?=$this->escapeHtmlAttr($this->facet['displayText']) ?>" data-count="<?=$this->facet['count'] ?>"
+        title="<?php if($this->facet['isApplied']): ?><?=$this->transEsc('applied_filter') ?> - <?=$this->transEsc('page_reload_on_deselect_hint', ['%%filter_name%%' => $this->facet['displayText']])?> <?=$this->escapeHtmlAttr($this->facet['displayText']) ?><?php else: ?><?=$this->transEsc('page_reload_on_select_hint', ['%%filter_name%%' => $this->facet['displayText']])?><?php endif;?>">
       <?=$displayText ?>
+      <span class="sr-only">
+        <?php if($this->facet['isApplied']): ?>(<?=$this->transEsc('applied_filter')?> - <?=$this->transEsc('page_reload_on_deselect_hint', ['%%filter_name%%' => $this->facet['displayText']])?> <?=$this->escapeHtmlAttr($this->facet['displayText']) ?>)<?php else: ?>(<?=$this->transEsc('page_reload_on_select_hint', ['%%filter_name%%' => $this->facet['displayText']])?>)<?php endif;?>
+      </span>
```

Comment:
This is quite verbose to put on every single facet. It might be good to put this text by the sidebar header or even near the top of each section so it's conveyed once. As it is now, this will make many of the links on the page unnecessarily wordy.

Reply:
You are absolutely right. Putting it at the top of the facets so it can be read once when users arrive there might be the best option.

Reply:
Thanks, @ckaz, do you have time to try for a revision, and to address some of the other feedback here? I also note there are some merge conflicts that need to be resolved -- let me know if you need any help with those.

It appears that we haven't set a milestone for this. Do you feel this is something we should add with a target of release 9? If so, we should try to expedite work on it so that we can finalize the new language strings, so that this doesn't block our ability to start work on translation for the new release.

Reply:
Yes, I will give this a shot on monday. How do I bring my branch to VuFind 9? I could check it out afresh based on the VF9-branch?

Reply:
You should be able to merge in the dev branch to get things up to date, but there will definitely be conflicts to resolve. If it's easier to just reapply changes to dev and then force-push to this branch, that's another viable approach.